relay-server/src/utils/managed_envelope.rs

```diff
@@ -158,7 +158,7 @@ impl ManagedEnvelope {
 
     /// Consumes itself returning the managed envelope.
     ///
-    /// This also releases the slot with [`SemaphorePermit`] and sets the internal context
+    /// This also releases the slot with [`OwnedSemaphorePermit`] and sets the internal context
```

Comment:
```suggestion
    /// This also releases the slot with [`SemaphorePermit`] and sets the internal context
```

Reply:
heh, seen this myself. Changed it already.  in [54dd9d3](https://github.com/getsentry/relay/pull/2096/commits/54dd9d36048537a64eacd4e94deb7828c5ae460b)